hashicorp-vault-pam/VaultAPI.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright 2022 Keyfactor
```

Comment:
copyright is out of date (update in any files updated with this release)

Reply:
Are we supposed to just update these to the current year whenever we modify something?

Reply:
Are the dates to be updated in all project files or just the ones being modified? There should be a tool to update the date if all files need updating.

Reply:
> Are the dates to be updated in all project files or just the ones being modified? There should be a tool to update the date if all files need updating.

MIkey, I just updated all 3 code files.  It's a small repo.